challenges/objects-arrays.js

```diff
@@ -89,37 +133,54 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
+// for(let i = 0; i < zooAnimals.length; i++) {
+//   animalNames.push(`Name: ${zooAnimals[i].animal_name}, Scientific: ${zooAnimals[i].scientific_name}.`);
+// }
+
+zooAnimals.forEach(item => {
+  animalNames.push(`Name: ${item.animal_name}, Scientific: ${item.scientific_name}.`);
+});
+
 console.log(animalNames);
 
-/* Request 2: .map()    
+/* Request 2: .map()
 
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
 
 const lowerCase = [];
-console.log(lowerCase); 
+zooAnimals.map((whatever) => {
+  lowerCase.push(whatever.animal_name.toLowerCase());
+});
+console.log(lowerCase);
 
-/* Request 3: .filter() 
+/* Request 3: .filter()
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
 const lowerPopulation = [];
+zooAnimals.filter((whatever) => {
```

Comment:
filter also creates a new array, so you don't need an empty array to push into and you don't need the if statement.